test/extended/networking/external_gateway.go

```diff
@@ -0,0 +1,46 @@
+package networking
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Ginkgo imports typically have an explicit prefix

```suggestion
	g "github.com/onsi/ginkgo"
```

Reply:
done